src/server.js

```diff
@@ -267,7 +267,7 @@ const validEmail = (email) => {
 }
 
 const validUsername = (username) => {
-	if (!/[a-zA-Z0-9_]/.test(username) || username === "") {
+	if (!(/^[a-zA-Z0-9_]*$/.test(username)) || username === "") {
```

Comment:
What did you do here?

Reply:
I think the regex verified if there was at least one alphanumerical character in the string. I changed it so that all the characters have to be alphanumerical.